core/src/test/scala/latis/util/FileUtilsSpec.scala

```diff
@@ -0,0 +1,24 @@
+package latis.util
+
+import org.scalatest.FlatSpec
+import org.scalatest.Matchers._
+import java.nio.file.Paths
+
+class FileUtilsSpec extends FlatSpec {
+  
+  /* 
+   * This is designed to find the file in the local file system, 
+   * a case where tests have failed before.
+   * Some testing environments may find it in a jar file. 
+   * If they do, then they didn't suffer from the original problem.
+   */
+  "The path resolver" should "find a file in the classpath" in {
+    val path = Paths.get("fdml.xsd")
+    assert(FileUtils.resolvePath(path).nonEmpty)
+  }
+  
+  "The path resolver" should "find a file in a jar in the classpath" in {
```

Comment:
The test output is strange unless you do this:

```suggestion
  it should "find a file in a jar in the classpath" in {
```